apps/electron-app/src/renderer/src/components/ui/context-menu-detector.tsx

```
@@ -0,0 +1,141 @@
+    }
+
+    // Check if it's a link
+    const linkElement = target.closest("a");
+    if (linkElement && linkElement.href) {
+      info.isLink = true;
+      info.linkUrl = linkElement.href;
+    }
+
+    // Get selected text
+    const selection = window.getSelection();
+    if (selection && selection.toString().trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)